crates/rooch/src/commands/account/commands/update.rs

```diff
@@ -25,67 +25,59 @@ pub struct UpdateCommand {
     mnemonic_phrase: String,
     #[clap(flatten)]
     pub context_options: WalletContextOptions,
-    /// Command line input of coin ids
-    #[clap(short = 'c', long = "coin-id", arg_enum)]
-    pub coin_id: CoinID,
+    /// Command line input of multichain ids
+    #[clap(short = 'i', long = "multichain-id")]
+    pub multichain_id: RoochMultiChainID,
```

Comment:
We do not need to support multichain_id for update command. Just remove it.

Reply:
Removed.